holoviews/plotting/bokeh/path.py

```diff
@@ -52,8 +52,17 @@ def _element_transform(self, transform, element, ranges):
                 else:
                     new_data.append(d)
             return np.array(new_data)
-        return np.concatenate([transform.apply(el, ranges=ranges, flat=True)
-                               for el in element.split()])
+
+        transformed = []
+        for el in element.split():
+            new_el = transform.apply(el, ranges=ranges, flat=True)
+            if len(new_el) == 1:
+                kdim_length = max([len(el[kdim]) for kdim in el.kdims])
```

Comment:
```suggestion
                kdim_length = len(el[el.kdims[0]])
```

Can the first dim be used for the length? If it needs to check all kdims, it would be nice to have a test for that.

Reply:
Yes, I think we can use the first kdim. Is it safe to assume that the kdims values at this point are equal length?